app/controllers/admin/students_controller.rb

```diff
@@ -1,20 +0,0 @@
-module Admin
-  class StudentsController < ResourceController
```

Comment:
I had put grove monitor on the students controller, and realized it would be better served on the students playlist controller. Also, realized that we would probably need to be able to CRUD students as a separate resource in the future.